pip/basecommand.py

```diff
@@ -105,17 +105,9 @@ def parse_args(self, args):
     def main(self, args):
         options, args = self.parse_args(args)
 
-        if options.quiet:
-            if options.quiet == 1:
-                level = "WARNING"
-            elif options.quiet == 2:
-                level = "ERROR"
-            else:
-                level = "CRITICAL"
-        elif options.verbose:
-            level = "DEBUG"
-        else:
-            level = "INFO"
+        verbosity = options.verbose - options.quiet
+        level_dict = {1: "DEBUG", -1: "WARNING", -2: "ERROR", -3: "CRITICAL"}
```

Comment:
Thanks for advice. I have not thought the > and < behavior, I will fix it soon.

But, does it make sense to allow users does `pip install -qqqqqqqq` ? Or we can stop user doing this strange thing, for example, throw an error?